### DISCUSSION TITLE
Monika cant read my gifts and its making me upset cause i cant gift her

### DISCUSSION THREAD

**Discussion by Ethan4Spirit** [Monika-After-Story/MonikaModDev#9314] [QA - tech support + general]
No matter what i do she still cant read my gifts and i dont get it i have watched every single tutorial and it doesnt help please just like help me i have done all the gifts i wanted but it still just doesnt want to work

**Comment by Wingdinggaster656** [comment]
Calm down when you are describing an issue, since it matters not how manic you are.

Would you take a screenshot to show me how you gifted?

**Comment by Ethan4Spirit** [comment]
> Calm down when you are describing an issue, since it matters not how manic you are.
> 
> Would you take a screenshot to show me how you gifted?

what do you mean im kinda confused

**Comment by Wingdinggaster656** [comment]
I mean after you gifted the gift, take a screenshot of your `/character` folder.

**Comment by Ethan4Spirit** [comment]
> I mean after you gifted the gift, take a screenshot of your `/character` folder.

oh ok

**Comment by Ethan4Spirit** [comment]
> I mean after you gifted the gift, take a screenshot of your `/character` folder.

For some reason whenever i gift her something then i look at the chr folder its just the chr files no gift file in there

**Comment by Wingdinggaster656** [comment]
I mean take a screenshot of your folder immediately after you created the gift file.

**Comment by Ethan4Spirit** [comment]
oki

**Comment by Ethan4Spirit** [comment]
umm how do u send a picture on pc? 
heh

**Comment by Ethan4Spirit** [comment]
Wait i dont know why but it worked

**Comment by Ethan4Spirit** [comment]
Im so sorry have a great day

**Comment by Wingdinggaster656** [comment]
That's fine. Enjoy your life.
